src/NBitcoin/Network.cs

```diff
@@ -195,7 +196,7 @@ public Consensus()
 
         public int SubsidyHalvingInterval { get; set; }
 
-        public Func<BlockHeader, uint256> GetPoWHash { get; set; } = h => h.GetHash();
+        public Func<NetworkOptions, BlockHeader, uint256> GetPoWHash { get; set; } = (n,h) => h.GetHash(n);
```

Comment:
I dont like this delegate I would rather have an abstract (or an override on NetworkOptions that returns the `POWHash()` then stratis can override with its own part.
We can leave it to another PR but please create an issue for this.

Reply:
https://github.com/stratisproject/StratisBitcoinFullNode/issues/888